src/bsmap/main.cpp

```diff
@@ -206,6 +206,7 @@ cerr<<"Usage:	bsmap [options]\n"
 		<<"       -o  <str>   output alignment file, BSP/SAM/BAM format, if omitted, the output will be written to STDOUT in SAM format.\n"
 		<<"\n  Options for alignment:\n"
 		<<"       -s  <int>   seed size, default=16(WGBS mode), 12(RRBS mode). min=8, max=16.\n"
+        <<"       -U          leave the output bam unsorted(reads will follow the order of the input fastq files), default="<<param.unsort<<"\n\n"		

```

Comment:
It is more about reporting as unsorted BAM, not alignment parameters. Move it to the reporting section. Ensure no extra newline.

Reply:
Solved